### PR TITLE
fix: add more debug output to jupyter command

### PR DESCRIPTION
**Pull request by bloodearnest**
It's hard to test this on different users setups, so make more debugging
output available.

### CODE REVIEW

File: opensafely/jupyter.py [hunk 1]
```diff
@@ -13,6 +13,18 @@
...
+# poor mans debugging because debugging threads on windows is hard
+if os.environ.get("DEBUG", False):
+    def debug(msg):
+        # threaded output for some reason needs the carriage return or else
+        # it doesn't reset the cursor.
+        sys.stderr.write("DEBUG: " + msg.replace("\n", "\r\n") + "\r\n")
+        sys.stderr.flush()
+else:
+    def debug(msg):
+        pass
```
Comment (ghickman) [inline opensafely/jupyter.py:25]: Given the function name and parameters are the same for both forks, is it worth pushing the if-block inside the function?

Something like:

```
def debug(msg):
    DEBUG = os.environ.get("DEBUG", False)
    if not DEBUG:
        return

    # true branch
    …
```

**Review by ghickman — APPROVED** [review]
One small thought on layout of the `debug` function, feel free to ignore!

File: opensafely/jupyter.py (hunk 1 above)
Comment (bloodearnest) [reply opensafely/jupyter.py:25]: Why do 20 `if` s if you can do one!

Comment (bloodearnest) [reply opensafely/jupyter.py:25]: Also, I plan to DTRT thing an add debug logging and -vvv switches at somepoint, so this is only temporary

**Pull request merged**